### PR TITLE
Update Fields - Sort of user document 

### PR DESCRIPTION
**Pull request by recca5p**

### CODE REVIEW

File: docs/user/fields/sort-by-field.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The *Sort by field* option is part of the fields settings, but is hidden by defa
 For more practical information, take a look at the following examples
 that showcase this functionality:
 
-- [Sort by Field](#sort-by-field)
+- [Sort by Field](#)
   - [Enabling Sort by Field](#enabling-sort-by-field)
   - [Ordering Support Cases by Priority](#ordering-support-cases-by-priority)
   - [Ordering String Dates in Chronological Order](#ordering-string-dates-in-chronological-order)
```
Comment (hainv-ohio) [inline docs/user/fields/sort-by-field.md:26]: Remove, we don't need this

File: docs/user/fields/sort-by-field.md [hunk 2]
```diff
@@ -62,4 +62,4 @@ data editor, and select your desired sorting.
 Once done, select **Update Field**. Your information will now be ordered
```
Comment (hainv-ohio) [inline docs/user/fields/sort-by-field.md:62]: Add 2 new lines here.
image

**Review by hainv-ohio — CHANGES_REQUESTED** [review]

File: docs/user/fields/sort-by-field.md (hunk 1 above)
Comment (recca5p) [reply docs/user/fields/sort-by-field.md]: Removed

File: docs/user/fields/sort-by-field.md (hunk 2 above)
Comment (recca5p) [reply docs/user/fields/sort-by-field.md:62]: Added

**Review by hainv-ohio — APPROVED** [review]

**Pull request merged**